src/libraries/System.Private.CoreLib/src/System/Char.cs

```diff
@@ -79,6 +79,7 @@ namespace System
         };
 
         // Return true for all characters below or equal U+00ff, which is ASCII + Latin-1 Supplement.
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
Can we instead fix inlining? This case is pretty basic.

Reply:
the only thing we can do here is to increase the ALWAYS_INLINE size which is currently 16 bytes of IL.
Also, we can save a bit in VM that a method wasn't inlined but its final codegen was small - but it won't help if IsDigit will be the first method with `IsLatin1` to compile.

Reply:
> two unknown calls 

Why is Span.Length "unknown"?

Reply:
> Why is Span.Length "unknown"?

For the inliner all calls are "unknown" unless they're [Intrinsic], I guess we can mark Span.Length with it just for the inliner's sake.

Reply:
For array length, we get `ldlen` in the IL but `{RO}Span<T>` lengths get a function call emitted (same as string.Length). Maybe we can recognize `get_Length` of `{RO}Span<T>`  and use `gtNewArrLen()` helper as done for `CEE_LDLEN` and `NI_System_String_get_Length` cases?

Currently these are some frequently used methods in template apps, some of which importer recognizes for optimizations and inlining, others are unknown:
```
# String
Named Intrinsic System.String..ctor: Not recognized
Named Intrinsic System.String.FastAllocateString: Not recognized
Named Intrinsic System.String.get_Chars: Recognized ✔️
Named Intrinsic System.String.get_Length: Recognized ✔️

# ROS<T>
Named Intrinsic System.ReadOnlySpan`1.CopyTo: Not recognized
Named Intrinsic System.ReadOnlySpan`1..ctor: Not recognized
Named Intrinsic System.ReadOnlySpan`1.get_Item: Recognized ✔️
Named Intrinsic System.ReadOnlySpan`1.get_Length: Not recognized
Named Intrinsic System.ReadOnlySpan`1.GetPinnableReference: Not recognized
Named Intrinsic System.ReadOnlySpan`1.op_Implicit: Not recognized
Named Intrinsic System.ReadOnlySpan`1.Slice: Not recognized

# Span<T>
Named Intrinsic System.Span`1..ctor: Not recognized
Named Intrinsic System.Span`1.Fill: Not recognized
Named Intrinsic System.Span`1.get_IsEmpty: Not recognized
Named Intrinsic System.Span`1.get_Item: Recognized ✔️
Named Intrinsic System.Span`1.get_Length: Not recognized
Named Intrinsic System.Span`1.op_Implicit: Not recognized
Named Intrinsic System.Span`1.Slice: Not recognized
Named Intrinsic System.Span`1.ToString: Not recognized
```

Reply:
@am11 inliner doesn't materialize IL into IR, it only operates with raw IL, so Span.get_Length is just a random call for it